src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -80,7 +81,30 @@ const defaultProps = {
 
 class ReimbursementAccountPage extends React.Component {
     componentDidMount() {
-        fetchFreePlanVerifiedBankAccount();
+        // We can specify a step to navigate to by using route params when the component mounts.
+        fetchFreePlanVerifiedBankAccount(this.getStepToOpenFromRouteParams());
+    }
+
+    componentDidUpdate(prevProps) {
+        const currentStep = lodashGet(
+            this.props,
+            'reimbursementAccount.achData.currentStep',
+            CONST.BANK_ACCOUNT.STEP.BANK_ACCOUNT,
```

Comment:
@marcaaron I have a question: Why do we need to add the default value here?

Reply:
It's been a while for me - but it would seem to imply that there's no `reimbursementAccount` data to pull the currentStep from. The "bank account" step is the first step in this flow so that's why we fall back to it.